migrations/2024-09-12-130204_drop-enable-nsfw/up.sql

```diff
@@ -0,0 +1,20 @@
+-- if site has enable_nsfw, set a default content warning
+UPDATE
+    site
+SET
+    content_warning = CASE WHEN local_site.enable_nsfw THEN
+        'nsfw'
```

Comment:
Nitpick: this context warning might look better if the NSFW acronym is uppercase.

Reply:
Changed